irohad/multi_sig_transactions/state/mst_state.hpp

```diff
@@ -148,6 +150,12 @@ namespace iroha {
      */
     bool contains(const DataType &element) const;
 
+    /// Get the amount of stored transactions.
```

Comment:
```suggestion
    /// Get the number of stored transactions over batches.
```
1. amount -> number or quantity
2. \+ "over batches"
https://www.bkacontent.com/commonly-confused-words-amount-number-and-quantity/